scripts/copy-carthage-frameworks.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+case "$PLATFORM_NAME" in
+    macosx) plat=Mac;;
+    iphone*) plat=iOS;;
+    watch*) plat=watchOS;;
+    appletv*) plat=tvOS;;
+    *) echo "error: Unknown PLATFORM_NAME: $PLATFORM_NAME"; exit 1;;
+esac
+
+for (( n = 0; n < SCRIPT_INPUT_FILE_COUNT; n++ )); do
+    VAR=SCRIPT_INPUT_FILE_$n
+    framework=$(basename "${!VAR}")
+    export SCRIPT_INPUT_FILE_$n="$SRCROOT"/Carthage/Build/$plat/"$framework".framework
+done
+
+/usr/local/bin/carthage copy-frameworks || exit
+
+for (( n = 0; n < SCRIPT_INPUT_FILE_COUNT; n++ )); do
+    VAR=SCRIPT_INPUT_FILE_$n
+    source=${!VAR}.dSYM
+    dest=${BUILT_PRODUCTS_DIR}/$(basename "$source")
```

Comment:
BTW, which dSYM is here? and what's for? carthage copy-framework seems will copy `FBSnapshotTestCase.framework.dSYM` under `CodeCoverage/Products/Debug-iphonesimulator/`